alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultConfigurationAccessor.java

```diff
@@ -325,6 +325,7 @@ public ConfigurationModel updateConfiguration(Long descriptorConfigId, Collectio
 
         List<FieldValueEntity> oldValues = fieldValueRepository.findByConfigId(descriptorConfigId);
         fieldValueRepository.deleteAll(oldValues);
+        fieldValueRepository.flush();
```

Comment:
It seems odd that we need to add these flushes throughout our code.

Reply:
Yeah, seems like a code smell, but it shouldn't cause an issue other than a minor performance hit.